src/renderer/config-blocks/ElementName.svelte

```diff
@@ -1,17 +1,16 @@
-<script context="module">
+<script lang="ts" context="module">
+  import type { ActionBlockInformation } from "./ActionBlockInformation.ts";
   // Component for the untoggled "header" of the component
   import RegularActionBlockFace from "./headers/RegularActionBlockFace.svelte";
   export const header = RegularActionBlockFace;
 
   // config descriptor parameters
-  export const information = {
+  export const information: ActionBlockInformation = {
     short: "sn",
     name: "ElementName",
-    rendering: "standard", //'hidden'
-    toggleable: false,
```

Comment:
why is toggleable attribute removed? ElementName block should always render opened, ready for editing!

Reply:
The property toggleable was present in ElementNames.svelte twice with different values. The unnecessary and incorrect one (above) was removed.